main/menu.js

```diff
@@ -33,6 +34,8 @@ function onPlayerClose () {
   getMenuItem('Increase Speed').enabled = false
   getMenuItem('Decrease Speed').enabled = false
   getMenuItem('Add Subtitles File...').enabled = false
+
+  thumbnail.showPlayerThumbnailBar()
```

Comment:
This really isn't the right place for this. It's not related to the menu. I'll move it.